Data/DynamicInputPack.cs

```diff
@@ -206,6 +206,17 @@ public ICommand EditTextureCommand
 
         private void EditTexture(DynamicInputTexture texture)
         {
+            //Automatically select suitable zoom.
+            if (texture.ScaleFactor == 1)
+            {
+                double scale = (texture.ImageHeight + texture.ImageWidth) / 2;
+                texture.ScaleFactor = 600 / scale;
+                if (texture.ScaleFactor > 1)
```

Comment:
`if (texture.ScaleFactory > 1.0)`

Reply:
This probably isn't necessary for the same reason I mentioned in the other comment.